Python/Python.sublime-syntax

```diff
@@ -395,6 +391,15 @@ contexts:
     - match: '@'
       scope: keyword.operator.matrix.python
 
+  allow-unpack-operators:
+    # Match unpacking operators, if present
+    - match: \*
+      scope: keyword.operator.unpacking.python
+    - match: \*\*
+      scope: keyword.operator.unpacking.python
```

Comment:
Currently, 3 or more stars in a row will be scoped as if valid, is that what we want?